adapters/ddf-adapter/src/main/java/com/connexta/ion/replication/adapters/ddf/rest/DdfRestClientFactory.java

```diff
@@ -70,7 +69,10 @@ public DdfRestClient createWithSubject(URL url) {
                 false,
                 false,
                 connectionTimeout,
-                receiveTimeout)
+                receiveTimeout,
+                System.getProperty("javax.net.ssl.certAlias", "localhost"),
```

Comment:
✏️ We might want to avoid retrieving the property for `System.getProperty()` each time we need it. Caching it maybe?